actors/evm/tests/contracts/TablelandTest.sol

```diff
@@ -0,0 +1,172 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.17;
+
+contract TablelandTest {
```

Comment:
Added all the functions required for encoding/decoding CBOR in this one contract so it is easier to test in this repo  with the current setup, without building with dependencies.

Reply:
Wow, I didn't think far enough down the line to see we'd need the coders here but makes sense. Is it possible to decode data returned from a query in any usable way?

Reply:
Yes, I think so. In the ping/pong example in the test case, we get back the string "pong" when we call the `ping` method on the actor. So it seems, we can decode arbitrary query results encoded as strings.